src/app/components/templates/ResultZabo/ResultTextZabo.tsx

```diff
@@ -5,84 +5,84 @@ import Link from 'next/link';
+          <div className="flex items-center justify-center gap-2">
+            <DefaultProfile className="h-10 w-10 md:h-9 md:w-9" />
+            <div className="flex flex-col gap-0 font-medium md:flex-row md:items-center md:gap-1">
+              <div className="text-base md:text-lg">
+                {searchQuery ? (
+                  <HighlightedText query={searchQuery}>
+                    {author.name}
+                  </HighlightedText>
+                ) : (
+                  author.name
+                )}
+              </div>
+              <div className="hidden text-base text-greyDark md:flex">·</div>
+              <div className="text-xs font-normal text-greyDark md:text-base md:font-medium">
+                {dayjs(createdAt).fromNow()}
+              </div>
             </div>
-          )}
-          <div className="text-start text-3xl font-bold">
-            {searchQuery ? (
-              <HighlightedText query={searchQuery}>{title}</HighlightedText>
-            ) : (
-              title
-            )}
           </div>
-
-          <div className="flex items-center gap-0.5">
-            <div className="text-lg font-bold">
-              {searchQuery ? (
-                <HighlightedText query={searchQuery}>
-                  {author.name}
-                </HighlightedText>
+          {currentDeadline && (
+            <div
+              className={`h-fit rounded-md ${
+                isClosed ? 'bg-greyDark' : 'bg-primary'
+              } px-2 py-1 text-sm text-white`}
+            >
+              {isClosed ? (
+                t('ddayPlus')
               ) : (
-                author.name
+                <Trans t={t} i18nKey={'zabo.timeLeft'}>
+                  {{ timeLeft: dayjs(currentDeadline).fromNow(true) }}
+                </Trans>
               )}
             </div>
-            {/* organization here (for futer update) */}
-          </div>
-
-          <Tags
-            tags={tags}
-            className="my-0.5"
-            searchQuery={searchQuery}
-            lng={lng}
-          />
-
-          <div className="line-clamp-4 text-ellipsis text-start text-sm font-medium">
-            {content ?? t('zabo.noContent')}
-          </div>
-
-          <div className="flex gap-0.5">
-            <div className="flex text-sm font-medium text-secondaryText">
-              <Trans t={t} i18nKey="zabo.dateView">
-                {{ date: dayjs(createdAt).tz().format('L') }}
-                <strong className="font-bold"> · {{ views }}</strong>
-              </Trans>
-            </div>
-          </div>
+          )}
+        </div>
+        <div className="text-xl font-semibold">
+          {searchQuery ? (
+            <HighlightedText query={searchQuery}>{title}</HighlightedText>
+          ) : (
+            title
+          )}
+        </div>
+        <div className="line-clamp-4 text-ellipsis text-start text-sm font-medium">
```

Comment:
```suggestion
        <div className="line-clamp-4 text-ellipsis text-start font-medium">
```
레귤러로 바꾸는 게 나을것같아요 sm은 너무 작아서..